File: BackEnd/src/main/java/com/unosquare/admin_core/back_end/entity/EventMessage.java
package com.unosquare.admin_core.back_end.entity;

import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.time.LocalDate;
import java.time.LocalDateTime;

@Data
@Entity
@NoArgsConstructor
@Table(name = "Event_Message")
public class EventMessage {

    @Id
    @SequenceGenerator(name="eventMessageSeq",sequenceName="event_message_event_message_id_seq", allocationSize = 1)
    @GeneratedValue(strategy=GenerationType.SEQUENCE, generator="eventMessageSeq")

    @Column(name = "event_message_id", unique = true, nullable = false)
    private int eventMessageId;

    @OneToOne()
    @JoinColumn(name = "eventId")
    private Event event;

    @OneToOne()
    @JoinColumn(name = "employee_id")
    private Employee employee;

    @Column(name = "last_modified")
    private LocalDateTime lastModified;

    @Column(name = "message")
    private String Message;

    @OneToOne(cascade=CascadeType.DETACH)
    @JoinColumn(name = "event_message_type_id")
    private EventMessageType eventMessageType;

    public EventMessage(int eventMessageId) {
        this.eventMessageId = eventMessageId;
    }
}

File: v2/AdminCore.DAL/Models/EventMessage.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace AdminCore.DAL.Models
{
  [Table("event_message")]
  public class EventMessage
  {
    [Key]
    [Column("event_message_id")]
    public int EventMessageId { get; set; }

    [Column("event_id")]
    public int EventId { get; set; }

    [Column("message")]
    public string Message { get; set; }

    [Column("event_message_type_id")]
    public int EventMessageTypeId { get; set; }

    [ForeignKey("EventMessageTypeId")]
    public EventMessageType EventMessageType { get; set; }

    [Column("last_modified")]
    public DateTime LastModified { get; set; }

    [Column("employee_id")]
    public int EmployeeId { get; set; }

    [ForeignKey("EmployeeId")]
    public Employee Employee { get; set; }
       
    [ForeignKey("EventId")]
    public Event Event { get; set; }
  }
}